packages/editor-ui/src/store.ts

```diff
@@ -883,9 +897,9 @@ export const store = new Vuex.Store({
 		},
 		pinDataSize: (state) => {
 			return state.workflow.nodes
-				.filter((node) => node.pinData)
+				.filter((node) => node.pinData && node.name !== state.activeNode)
```

Comment:
Can we simplify?

```ts
pinDataSize: (state) => {
	return state.workflow.nodes
		.reduce((acc, node) => {
			if (typeof node.pinData !== 'undefined' && node.name !== state.activeNode) {
				acc += stringSizeInBytes(node.pinData);
			}
			return acc;
		}, 0);
```

Otherwise, for proper type flow:
```ts
.filter(
	(node): node is INodeUi & { pinData: IDataObject } => {
		return typeof node.pinData !== 'undefined' && node.name !== state.activeNode
	})
.reduce((acc, node) => {
	acc += stringSizeInBytes(node.pinData);
	return acc;
}, 0);
```

Reply:
Simplified as suggested. Thank you!